src/modules/shared/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import NavDropdown from "react-bootstrap/NavDropdown";
+import Nav from "react-bootstrap/Nav";
+const DropdownMenu = () => {
+  return (
+    <Nav className="">
+      <NavDropdown
+        id="nav-dropdown-dark-example"
+        title={
+          <svg
```

Comment:
If we want to remove the clutter from the svg element. We can make it as a JSX component and include it as an icon. We can use this as well to convert it, but it is a straightforward process .
https://svg2jsx.com/